_workinggroups/pyhep.md

```diff
@@ -20,7 +25,7 @@ and by means of the following communication channels:
 * [Gitter channel PyHEP](https://gitter.im/HSF/PyHEP){:target="_pyhep_gitter_channel} for any informal exchanges.
 * [GitHub repository of resources](https://github.com/hsf-training/PyHEP-resources){:target="_pyhep_resources_repo"},
```

Comment:
Btw, what does this target specification do? If I look at the `target` attribute of hyperlinks, this would only make sense if you have iframes of that name...

Reply:
Not sure (it does anything). It was added when the page was first created in PR #397.

Reply:
Fair enough to have better consistency.

As far as Twitter is concerned, I'm no expert but do remember there is also https://twitter.com/pyhepconf?lang=en, which we link to for the workshops. This one has more and more recent updates. What to do?

Reply:
Good point. I've updated to that now.

Reply:
Thanks.